extensions/barrage/src/main/java/io/deephaven/extensions/barrage/table/BarrageTable.java

```diff
@@ -250,7 +252,9 @@ public void handleBarrageError(Throwable t) {
 
     @Override
     public void run() {
-        refreshEntry.onUpdateStart();
+        if (refreshEntry != null) {
```

Comment:
We talked about maybe using an inner class so we could use `InstrumentedUpdateSource`. I don't feel very strongly about that, but it's OK if you want to. Same for `TimeTable`.